python/tests/test_c_schema_view.py

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from nanoarrow.c_lib import c_schema_view
+from nanoarrow.c_schema import c_schema_view
```

Comment:
Nit: this is also available from the top-level namespace, so if changing this, maybe directly to that?